package.json

```diff
@@ -17,50 +17,25 @@
   "types": "./dist/main.d.ts",
   "scripts": {
     "dev": "vite",
-    "preview": "vite preview",
     "build": "npm run lint && vite build",
     "build:lib": "vite build",
     "lint": "eslint . --max-warnings=0 && vue-tsc --noEmit",
     "prepublishOnly": "npm run build",
     "prepare": "npm run build",
     "test:unit": "vitest run --dom",
-    "cy:e2e:headless": "cypress run --e2e",
-    "cy:e2e": "cypress run --e2e --headed",
-    "test:e2e": "start-server-and-test dev http://localhost:8080 cy:e2e",
-    "test:e2e:headless": "start-server-and-test dev http://localhost:8080 cy:e2e:headless",
-    "web:open": "vite --open",
```

Comment:
should not be remove, is used to launch local app to test components

Reply:
Ah, yeah. I kept the `dev` but was a bit quick with the `web:open`.